Documentation/ThirdParties/Customers.md

```diff
@@ -0,0 +1,28 @@
+# Third Party Extension Support for Customers
```

Comment:
`Third party` if you're using it as a noun, `third-party` if you're using it as an adjective.  probably should be the later for most instances on this page.